zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/RequestMapper.java

```diff
@@ -787,6 +795,41 @@ public static Either<ProblemDetail, TenantDTO> toTenantUpdateDto(
                 tenantUpdateRequest.getDescription()));
   }
 
+  public static Either<ProblemDetail, AdHocSubprocessActivityQuery> toAdHocSubprocessActivityQuery(
```

Comment:
Currently validating that all filter properties are required - restrictions will be removed in #28432.